lib/galaxy/job_metrics/formatting.py

```diff
@@ -10,9 +10,11 @@ def format(self, key, value):
 
 def seconds_to_str(value):
     """Convert seconds to a simple simple string describing the amount of time."""
+    mins, secs = divmod(value, 60)
+    hours, mins = divmod(mins, 60)
     if value < 60:
-        return "%s seconds" % round(value, 2)
+        return "%s seconds" % secs
     elif value < 3600:
-        return "%s minutes" % round(value / 60, 2)
+        return "%s minutes" % mins
     else:
-        return "{} hours and {} minutes".format(value // 3600, round((value % 3600) / 60, 2))
+        return "%s hours and %s minutes" % (hours, mins)
```

Comment:
Looks good, thank you!
In dev forward (so, releases that are python 3.6+) we should standardize these on f-strings.
```suggestion
        return f"{hours} hours and {mins} minutes"
```